iree/compiler/Dialect/VM/Target/C/TranslationFlags.cpp

```diff
@@ -0,0 +1,58 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "iree/compiler/Dialect/VM/Target/C/TranslationFlags.h"
+
+#include "llvm/Support/CommandLine.h"
+
+namespace mlir {
+namespace iree_compiler {
+namespace IREE {
+namespace VM {
+
+static llvm::cl::opt<CCodeOutputFormat> outputFormatFlag{
+    "iree-vm-c-module-output-format",
+    llvm::cl::desc("Output format the c module is written in"),
```

Comment:
Maybe
```suggestion
    llvm::cl::desc("Output format the C module is written in"),
```
or even
```suggestion
    llvm::cl::desc("Output format used to write the C module"),
```
?